README.md

```diff
@@ -54,7 +54,16 @@ Thank you for your understanding! ^^
 * GraphicsMagick or ImageMagick
 * `make` and `g++` (for building npm binaries, *I had to do [this](https://github.com/fedwiki/wiki/issues/46) and [this](https://www.digitalocean.com/community/questions/node-gyp-rebuild-fails-on-install)*)
 
-## Setup
+## Setup (with Docker, for development environment)
+
+* Instal Docker Client via https://www.docker.com/
```

Comment:
would it be too specific to use that URL instead: https://www.docker.com/community-edition ?
(as the first download link I found on the homepage was for the entreprise edition, and I believe that the CE is enough.)